automation/project/Settings.scala

```diff
@@ -26,7 +26,8 @@ object Settings {
   )
 
   val testSettings = List(
-    testOptions in Test += Tests.Argument("-oF")
+    testOptions in Test += Tests.Argument("-oF"),
+    parallelExecution in Test := true
```

Comment:
this makes scalatest suites run in parallel, but LeonardoSpec is one giant suite, so it doesn't actually win us anything

Reply:
(see GAWB-2953 for more details on how to get the individual tests running in parallel)

Reply:
cool